pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-hadoop/src/main/java/org/apache/pinot/plugin/ingestion/batch/hadoop/HadoopSegmentGenerationJobRunner.java

```diff
@@ -143,13 +141,6 @@ public void run()
       PinotFSFactory.register(pinotFSSpec.getScheme(), pinotFSSpec.getClassName(), new PinotConfiguration(pinotFSSpec));
     }
 
-    //Get pinotFS for input
```

Comment:
Let's not move this. We want to throw exception before creating output dir when the input is not valid. We may move the list files logic here

Reply:
Updated